### PR TITLE
Make files cljc

### PR DESCRIPTION
**Pull request by joda**
Would like to use this in Clojurescript

### CODE REVIEW

**Comment by joda** [comment]
Added a deps.edn to make sure the tests ran in cljs, doesn't need to be merged if not wanted

File: src/zippo/core.cljc [hunk 1]
```diff
@@ -149,7 +149,8 @@
     ;; MapEntry doesn't support meta
     (map-entry? node)
     (let [[k v] children]
-      (new clojure.lang.MapEntry k v))
```
Comment (igrishaev) [inline src/zippo/core.cljc:152]: This one is tricky as it mixes with a regular vector. Let me think for a while...

**Comment by igrishaev** [comment]
Well, I did it in a separate PR: https://github.com/igrishaev/zippo/pull/2/files

It was a bit more tricky than I expected. Also, I added utilities to run cljs tests. I've just pushed the 0.1.3 version of the library with ClojureScript support.

**Comment by igrishaev** [comment]
You'll need at least version 1.9.542 of the ClojureScript compiler.

**Comment by joda** [comment]
Thanks a lot!

**Pull request closed without merging**